scrapers/SweetFemdom.yml

```diff
@@ -0,0 +1,23 @@
+name: "sweetfemdom"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sweetfemdom.com/tour/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3/text()
+      Date:
+        selector: //div[@class="videoInfo clear"]/p[1]/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Details: //div[@class="videoDetails clear"]/p/text()
```

Comment:
the details are a little more complex to get right
not only are there scenes with more paragraphs `https://sweetfemdom.com/tour/trailers/Randy-Moore-Sex-Therapist.html` `https://sweetfemdom.com/tour/trailers/Pegging-olympic-training-with-pepper-hart.html` but also some of the text isn't in the `<p>`
the safer selector seems to be 
```yaml
      Details:
        selector: //div[@class="videoDetails clear"]
        postProcess:
          - replace:
              - regex: ".*\t" # remove the h3 title and tab (\t) from description
                with: ""

```